options.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+ 
+add_action( 'admin_menu', 'add_plugin_page' );
+add_action( 'admin_init', 'page_init' );
+
+// Add the settings page to the dashboard menu
+function add_plugin_page()
+{
+    add_options_page(
+        'UIO Settings', 
+        'User Interface Options', 
+        'manage_options', 
+        'uio-setting-admin', 
+        'create_admin_page'
+    );
+}
+
+// Register the settings and define the form data
+function page_init()
+{        
+  register_setting( 'uio_option_group', 'uio_template_selector' );
+  register_setting( 'uio_option_group', 'uio_toc_selector' );
+
+    add_settings_section(
+        'setting_section_id', // ID
+        'Settings', // Title
+        'print_section_info', // Callback
+        'uio-setting-admin' // Page
+    );  
+
+    add_settings_field(
+        'uio_template_selector', // ID
+        'Where should we put <em>UIO</em>?', // Title
```

Comment:
Shall the plugin allow the localization of texts used in it? Same as strings used in line 52, 67, 68, 78, 83. 

A tutorial of how to do it - http://premium.wpmudev.org/blog/localize-a-wordpress-plugin-and-make-it-translation-ready/